test/binaryen.js/fatal.js

```diff
@@ -1,4 +1,8 @@
-var wasm = new Binaryen.Module()
-wasm.addFunctionType("vI", Binaryen.i32, []);
-// It will cause a fatal error to try to add the same name a second time
-wasm.addFunctionType("vI", Binaryen.i32, []);
+function test(Binaryen) {
```

Comment:
Is the `Binaryen` argument here different to the `Binaryen` in the global scope?   If not then maybe we don't need it?  Or at least maybe call it something else so its more obvious what it doesn't shadow the global one?

Reply:
It's the same as the global one, yeah. Mostly a case of "why not" if we'd ever want to test multiple variants, like a fastcomp and an upstream one, but can as well remove the arguments since we don't do that.